algo/src/main/java/org/neo4j/graphalgo/TriangleCountProc.java

```diff
@@ -0,0 +1,152 @@
+package org.neo4j.graphalgo;
+
+import org.neo4j.graphalgo.api.Graph;
+import org.neo4j.graphalgo.core.GraphLoader;
+import org.neo4j.graphalgo.core.ProcedureConfiguration;
+import org.neo4j.graphalgo.core.utils.Pools;
+import org.neo4j.graphalgo.core.utils.ProgressLogger;
+import org.neo4j.graphalgo.core.utils.ProgressTimer;
+import org.neo4j.graphalgo.core.utils.TerminationFlag;
+import org.neo4j.graphalgo.exporter.AtomicIntArrayExporter;
+import org.neo4j.graphalgo.exporter.IntArrayExporter;
+import org.neo4j.graphalgo.impl.ParallelTriangleCount;
+import org.neo4j.graphalgo.results.AbstractResultBuilder;
+import org.neo4j.kernel.api.KernelTransaction;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.logging.Log;
+import org.neo4j.procedure.*;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author mknblch
+ */
+public class TriangleCountProc {
+
+    public static final String DEFAULT_WRITE_PROPERTY_VALUE = "triangles";
+    @Context
+    public GraphDatabaseAPI api;
+
+    @Context
+    public Log log;
+
+    @Context
+    public KernelTransaction transaction;
+
+    @Procedure("algo.triangles.stream")
+    @Description("CALL algo.triangles.stream({nodeQuery:'labelName', relationshipQuery:'relationshipName', concurrency:8}) " +
```

Comment:
can we move the label + rel-type to regular parameters like we have in the other procedures?